src/app/participant-page/participant-page.component.html

```diff
@@ -524,6 +524,13 @@ <h3 class="Color--warn Line--Break">Lost to Follow-Up</h3>
                       <tr>
                         <td class="TD--Padding-Left Width--50">{{sample.receiveDate === 0 ? "-" : sample.receiveDate | date:'medium'}}</td>
                       </tr>
+                      <hr style="border-color: lightgray">
```

Comment:
Can we please hide the Deactivation part, when deactivatedDate === 0 (deactivation doesn't happen that often…)

Reply:
okay

Reply:
It's done